### PR TITLE
Add -k option when using curl to download google test

### PR DESCRIPTION
**Pull request by charsyam**
Add -k option when using curl to download google test

### CODE REVIEW

**Comment by sgolemon** [comment]
```
 -k, --insecure
              (SSL)  This  option  explicitly allows curl to perform "insecure" SSL connections and transfers. All
              SSL connections are attempted to be made secure by using the  CA  certificate  bundle  installed  by
              default. This makes all connections considered "insecure" fail unless -k, --insecure is used.
```

Is there a particular reason you want to open homebrew installations up to man-in-the-middle attacks?

**Comment by charsyam** [comment]
@sgolemon 
Yes. It has weak point from MITM. but -k option is almost about user convenience.

**Comment by sgolemon** [comment]
Let me rephrase.  *.googlecode.com is a verifiable certificate.  Why would you suspend verification of a verifiable certificate?

```
* SSL connection using ECDHE-RSA-AES128-GCM-SHA256
* Server certificate:
* 	 subject: C=US; ST=California; L=Mountain View; O=Google Inc; CN=*.googlecode.com
* 	 start date: 2015-06-03 09:49:55 GMT
* 	 expire date: 2015-09-01 00:00:00 GMT
* 	 subjectAltName: googletest.googlecode.com matched
* 	 issuer: C=US; O=Google Inc; CN=Google Internet Authority G2
* 	 SSL certificate verify ok.
```

**Comment by charsyam** [comment]
@sgolemon Thank you for your kind answer :), Actually, My macbook's environment is wrong.
In my case, I encountered some "SSL certificate problem".
after your answer, I requested some tests to my friends and I realized it's all my fault :)
Thank agian :)

**Comment by sgolemon** [comment]
Glad we got it sorted out! :)
